.github/workflows/draft-new-release.yml

```diff
@@ -0,0 +1,88 @@
+name: Draft New Release
+
+on: workflow_dispatch
+
+jobs:
+  draft-new-release:
+    name: Draft New Release
+    runs-on: ubuntu-latest
+
+    # Only allow release stakeholders to initiate releases
+    if: (github.ref == 'refs/heads/develop' || startsWith(github.ref, 'refs/heads/hotfix/')) && (github.actor == 'ItsSudip' || github.actor == 'krishna2020' || github.actor == 'utsabc' || github.actor == 'shrouti1507') &&  (github.triggering_actor == 'ItsSudip' || github.triggering_actor == 'krishna2020' || github.triggering_actor == 'utsabc' || github.triggering_actor == 'shrouti1507')
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.5.0
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node
+        uses: actions/setup-node@v3.6.0
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'npm'
+
+      - name: Install Dependencies
+        env:
+          HUSKY: 0
+        run: |
+          npm ci
+
+      # In order to make a commit, we need to initialize a user.
+      # You may choose to write something less generic here if you want, it doesn't matter functionality wise.
+      - name: Initialize Mandatory Git Config
+        run: |
+          git config --global user.name "GitHub Actions"
+          git config --global user.email "noreply@github.com"
+
+      # Calculate the next release version based on conventional semantic release
+      - name: Create Release Branch
+        id: create-release
+        env:
+          HUSKY: 0
+        run: |
+          source_branch_name=${GITHUB_REF##*/}
+          release_type=release
+          grep -q "hotfix/" <<< "${GITHUB_REF}" && release_type=hotfix-release
+          git fetch origin master
+          git fetch --tags origin
+          git merge origin/master
+          current_version=$(jq -r .version package.json)
+
+          npm run release -- --skip.commit --skip.tag --skip.changelog
```

Comment:
The `release` script is not available in `package.json`.

Reply:
added